docs/src/components/FeedbackCallout.js

```diff
@@ -14,26 +14,24 @@ export default function FeedbackCallout({ componentName }: Props): Node {
   const link = `${BASE_LINK}${componentName}`;
 
   return (
-    <Box marginLeft={-1} marginRight={-1}>
```

Comment:
`marginLeft` and `marginRight` are no longer supported on `Box` so this didn't do anything

Reply:
weird that Flow wasn't complaining about this given that we removed those props a while ago…